pages/linux/sacctmgr.md

```diff
@@ -15,6 +15,6 @@
 
 `sacctmgr add account {{account_name}} cluster={{cluster_of_account}}`
 
-- Show details of user/association/cluster/account:
+- Show details of user/association/cluster/account using a spcific format:
 
-`sacctmgr show {{user/association/cluster/account}}`
+`sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`
```

Comment:
oh, we don't use `[` to mark optional stuff.
```suggestion
`sacctmgr show {{user|association|cluster|account}} format="Accout%10" format="GrpTRES%30"`
```
Is this a valid example with 2 format arguments?

Reply:
It is valid. I use such commands to format my output.